src/main/docs/guide/introduction.adoc

```diff
@@ -33,6 +33,10 @@ The link:{micronautapi}discovery/DiscoveryClient[DiscoveryClient] is fairly simp
 
 Both methods return rs:Publisher[] instances since the operation to retrieve service ID information may result in a blocking network call depending on the underlying implementation.
 
+The link:{micronautapi}discovery/DiscoveryClient[DiscoveryClient] implementations all make use of the https://docs.micronaut.io/latest/guide/index.html#clientAnnotation[Declarative HTTP Client], thus you must also add one of the HTTP Client implementations as an explicit dependency:
```

Comment:
can you link to both Netty HTTP client and JDK HTTP Client.

Reply:
Done in [84f1284](https://github.com/micronaut-projects/micronaut-discovery-client/pull/609/commits/84f12847738223e12801bcf644e8e51e21e716f8)